src/applications/personalization/dashboard/tests/e2e/messaging-error.cypress.spec.js

```diff
@@ -1,13 +1,13 @@
 import { mockUser } from '@@profile/tests/fixtures/users/user';
-
+import { Paths } from '../../../../mhv/secure-messaging/tests/e2e/utils/constants';
 import ERROR_400 from '~/applications/personalization/dashboard/utils/mocks/ERROR_400';
 import vamcErc from '../fixtures/vamc-ehr.json';
 
 describe('MyVA Dashboard - Messaging', () => {
   describe('when there is an error fetching the inbox data', () => {
     beforeEach(() => {
       cy.login(mockUser);
-      cy.intercept('GET', '/my_health/v1/messaging/folders', {
+      cy.intercept('GET', Paths.INTERCEPT.MESSAGE_FOLDERS, {
         statusCode: 400,
         body: ERROR_400,
       });
```

Comment:
We should revert this particular change. 

Unfortunately this will break the idea that we should not be importing from other applications in order to leverage isolated app builds. I generally agree with keeping all the paths in one place and consolidating things into constants, so if you really wanted to do this I would recommend moving the constants to a place within the platform folder. That way the constants can be shared between any applications and will not break isolated app builds. 

[> More info on isolated app builds](https://depo-platform-documentation.scrollhelp.site/developer-docs/isolated-application-builds)

Reply:
I updated the code as per the review comment.